app/services/symphony_client.rb

```diff
@@ -79,20 +70,17 @@ def not_needed_after(hold_key:, fill_by_date:, session_token:)
                           })
   end
 
-  def renew_items(user, checkouts)
-    checkouts.each_with_object(success: [], error: []) do |checkout, status|
-      response = renew_item_request(checkout.resource,
-                                    checkout.item_key,
-                                    headers: { 'x-sirs-sessionToken': user.session_token })
-
-      case response.status
-      when 200
-        status[:success] << { renewal: checkout, sirsi_response: nil }
-      else
-        Rails.logger.error("Renewal (#{checkout.item_key}): #{response.body}")
-        status[:error] << { renewal: checkout, sirsi_response: (renewal_error_message(response) || '') }
-      end
-    end
+  def renew(resource:, item_key:, session_token:)
+    authenticated_request('/circulation/circRecord/renew',
+                          headers: { 'x-sirs-sessionToken': session_token },
+                          method: :post, json: {
+                            item: {
+                              resource: resource,
+                              key: item_key
+                            }
+                          }, params: {
+                            includeFields: 'circRecord{*}'
+                          })
```

Comment:
This is great too, that it's singular it can be simpler and error handling can be renewal job's concern. I like making symphony client "dumber"